### PR TITLE
Add Assert Shortcodes

### PR DESCRIPTION
**Pull request by anthony-murphy**
This just puts shortcodes in place. I'll push a PR for automation next. For now people should continue to just add normal messages to asserts, and the automation will convert them later

related #5409  #5314

### CODE REVIEW

**Comment by curtisman** [comment]
Can you describe what automation is going to do?

**Comment by anthony-murphy** [comment]
I'm writing a very simple script based on ts-morph. I want to integrate it into the release process, so before we release we run it. I can also add a test to validate that we have not duplicate tags, takes less than a second to run. will also be avaialbe for people to run manually if they want to tag their asserts at checkin time

**Comment by curtisman** [comment]
Will the source map be correct?

File: packages/dds/cell/src/cell.ts [hunk 1]
```diff
@@ -287,7 +287,7 @@ export class SharedCell<T extends Serializable = any> extends SharedObject<IShar
             if (local) {
                 const messageIdReceived = localOpMetadata as number;
                 assert(messageIdReceived !== undefined && messageIdReceived <= this.messageId,
-                    "messageId is incorrect from from the local client's ACK");
+                    "s_00" /* messageId is incorrect from from the local client's ACK */);
```
Comment (curtisman) [inline packages/dds/cell/src/cell.ts:290]: Are the checked in version be short code?

Comment (curtisman) [inline packages/dds/cell/src/cell.ts:290]: Why use string for short code? Can the short code be just numbers?

Comment (christiango) [reply packages/dds/cell/src/cell.ts:290]: From a bundle size perspective, I suspect base64 encoded numbers would be the biggest bang for our buck

Comment (curtisman) [reply packages/dds/cell/src/cell.ts:290]: Yeah, on the flip side, I am thinking of runtime memory size would be better with numbers.

Comment (anthony-murphy) [reply packages/dds/cell/src/cell.ts:290]: i planned to commit them to keep them unique and constant without needing a ton of tracking, currently the tagging script is stateless

Comment (anthony-murphy) [reply packages/dds/cell/src/cell.ts:290]: in terms of bundle size strings have a higher information density, could do hex numbers i guess which is a bit better

Comment (anthony-murphy) [reply packages/dds/cell/src/cell.ts:290]: i wasn't really concerned with in memory size

Comment (anthony-murphy) [reply packages/dds/cell/src/cell.ts:290]: currently they are radix 36 number strings, with supports 1296 unique codes for 2 characters, if and when we hit 3 chars, it goes up to 46656. we are not even very close the to two char limit (394), but if we ever cross that we should never hit the 3 char limit 46656. the current implementation  is a good balance between information density and complexity, `num.tostring(36)`

Comment (anthony-murphy) [reply packages/dds/cell/src/cell.ts:290]: if we care about memory size we can move to hex, we'll need 3 chars, which is probably fine, and but will also require code change

**Comment by anthony-murphy** [comment]
closing this. will come back as the automation goes in. thanks for the feedback!
